### PR TITLE
SNAP-860 Fix Hispanic/Latino mapping

### PR DESCRIPTION
**Pull request by pokharelmanish**
### Jira Story

- Fix Hispanic/Latino mapping https://osi-cwds.atlassian.net/browse/SNAP-860

## Description
Fix Hispanic/Latino frontend mapping.

Before:
screen shot 2019-01-22 at 10 38 12 am

After:
screen shot 2019-01-22 at 10 37 24 am

## Tests
- [x] I have included unit tests 
- [x] I have included feature tests 
- [ ] I have included other tests 
- [ ] I have NOT included tests 

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have ran all tests for the project.
- [ ] I have ran lint/rubocop check for the changed/new files.
- [ ] My code is in a stable state ready to be deployable, but not necessarily complete.
- [ ] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by pokharelmanish** [comment]
retest his please

**Comment by pokharelmanish** [comment]
retest this please

**Comment by pokharelmanish** [comment]
retest this please

File: app/javascript/reducers/peopleFormReducer.js [hunk 1]
```diff
@@ -45,8 +45,8 @@ const buildRaceDetails = (races = []) => races.reduce((racesValue, {race, race_d
   [race]: {value: race_detail},
 }), {})
 
-const buildEthnicity = (ethnicity = {}) => {
-  const {hispanic_latino_origin = null, ethnicity_detail = []} = ethnicity
+const buildEthnicity = (ethnicity = []) => {
```
Comment (money) [inline app/javascript/reducers/peopleFormReducer.js]: Why the design change? Wouldn’t it be safer to use the key value hash object?

Comment (pokharelmanish) [reply app/javascript/reducers/peopleFormReducer.js]: Good question. The backend JSON response was changed. Before it was an object but now its an array of object.

**Comment by tparkerosi** [comment]
I agree with will here. I only breifly looked at this issue so I don't know all the details, but ethnicity should be an object since it appears to return an object with properties and an array. The contract must be correct. We should spin off a story to correct this.

**Pull request merged**